css/default.css

```diff
@@ -31,8 +31,12 @@ span.dependency_upgrade		{ color: orange; }
 
 tr.bugnote .bugnote-note { background-color: #e8e8e8; color: #000000; width: 75%; vertical-align: top; }
 
+tr.bugnote > td.bugnote-note.bugnote-private { background-color: #fcf8e3;}
```

Comment:
This rule is overqualified, and anyway redundant considering the _important_ flag on the more generic rule below. Please remove it.